rules/terraformrules/terraform_empty_list_equality.go

```diff
@@ -0,0 +1,88 @@
+package terraformrules
+
+import (
+	"log"
+
+	"github.com/hashicorp/hcl/v2"
+	"github.com/hashicorp/hcl/v2/hclsyntax"
+	"github.com/terraform-linters/tflint/tflint"
+)
+
+// TerraformEmptyListCheckRule checks whether is there a comparison with an empty list
+type TerraformEmptyListCheckRule struct{}
```

Comment:
This and other related identifiers should be EmptyListEqualityRule

Reply:
done